docs/pages/01-usage.md

```diff
@@ -0,0 +1,87 @@
+## Installation and usage
+
+### Installing
+
+Install with npm:
+
+```shell
+$ npm install animate.css --save
+```
+
+with yarn:
+
+```shell
+$ yarn add animate.css
+```
+
+or add it directly to your webpage using a CDN:
+
+```html
+<head>
+  <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.7.2/animate.min.css"
+  />
+</head>
+```
+
+### Basic usage
+
+After installing Animate.css, add the class `animate__animated` to an element, along with any of the [animation names](#attention_seekers) (don't forget the `animate__` prefix!):
+
+```html
+<h1 class="animate__animated animate__bounce">An animated element</h1>
+```
+
+That's it! You've got a CSS animated element. Super!
+
+> Animations can improve the UX of an interface, but keep in mind that they can also get in the way of your users! Please read the [best practices](#best-practices) and [caveats](#caveats) sections to bring your web-things to life in the best way possible.
+
+#### Using `@keyframes`
+
+Even though the library provides you a few helper classes like the `animated` class to get you up running quickly, you can use the provided animations `keyframes` directly. This provides a flexible way to use Animate.css with your current projects without having to refactor your HTML code.
+
+Example:
+
+```css
+.my-element {
+  display: inline-block;
+  margin: 0 0.5rem;
+
+  animation: bounce; /* referring directly to the animation's @keyframe declaration */
+  animation-duration: 2s; /* don't forget to set a duration! */
+}
+```
+
+Be aware that some animations are dependent on the `animation-timing` property set on the animation's class. Changing, or not declaring it might harm the animation.
```

Comment:
```suggestion
Be aware that some animations are dependent on the `animation-timing` property set on the animation's class. Changing or not declaring it might lead to unexpected results.
```